src/constants/tokenlists.ts

```diff
@@ -1,49 +1,74 @@
-// Compliant list for exchange
-export const TOKEN_LIST_DEFAULT =
-  'https://storageapi.fleek.co/balancer-team-bucket/assets/listed.tokenlist.json';
-
-// Extended list to include LBP tokens
-export const VETTED_TOKEN_LIST =
-  'https://storageapi.fleek.co/balancer-team-bucket/assets/vetted.tokenlist.json';
-
-export const TOKEN_LISTS: string[] = [
-  TOKEN_LIST_DEFAULT,
-  'ipns://tokens.uniswap.org',
-  'tokenlist.zerion.eth',
-  'tokens.1inch.eth',
-  'tokenlist.aave.eth',
-  'https://tokens.coingecko.com/uniswap/all.json',
-  'https://umaproject.org/uma.tokenlist.json'
-];
+import useConfig from '@/composables/useConfig';
+
+const { env, networkConfig } = useConfig();
 
 export const ETHER = {
-  id: 'ether',
-  name: 'Ether',
-  address: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
-  symbol: 'ETH',
-  decimals: 18,
-  logoURI:
-    'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2/logo.png'
+  ...networkConfig.nativeAsset,
+  id: 'ether'
```

Comment:
Is this ID used anywhere? If not maybe we can get rid of the ETHER definition in this file altogether? I feel like it doesn't belong here at all.

Reply:
![image](https://user-images.githubusercontent.com/15848336/122945256-5d4c2000-d370-11eb-9882-458b057c4496.png)

It looks like we use it to allow having app.balancer.fi/ether/0x123... redirect to app.balancer.fi/0xee...ee/0x123... This behaviour seems a little odd to me (it's kinda a weird halfway step to readable deeplinks) so I think it's safe to remove but didn't want to start carving out behaviour.

Reply:
Yeah this is kind of weird behaviour but it would be good to maintain those deeplinks in case they're being used. Perhaps we can add a user friendly `deeplinkId` to the native asset config and then just reference that here in `populateInitialTokens`?

Reply:
Sure thing, I've made this change.

I don't think it would be right to remove `ETHER` entirely in this PR as it's spread out over the codebase so should probably be handled separately.